lib/matplotlib/tests/test_axes.py

```diff
@@ -743,7 +743,7 @@ def test_pcolormesh_datetime_axis():
     fig.subplots_adjust(hspace=0.4, top=0.98, bottom=.15)
     base = datetime.datetime(2013, 1, 1)
     x = np.array([base + datetime.timedelta(days=d) for d in range(21)])
-    y = np.arange(21)
+    y = range(21)
```

Comment:
Oh, that's nice to know (I don't do py3k regularly yet because of work
requirements)

On Wed, Jul 1, 2015 at 1:49 PM, Antony Lee <notifications@github.com> wrote:

> In lib/matplotlib/tests/test_axes.py
> <https://github.com/matplotlib/matplotlib/pull/4573#discussion_r33704379>:
>
> > @@ -743,7 +743,7 @@ def test_pcolormesh_datetime_axis():
> >      fig.subplots_adjust(hspace=0.4, top=0.98, bottom=.15)
> >      base = datetime.datetime(2013, 1, 1)
> >      x = np.array([base + datetime.timedelta(days=d) for d in range(21)])
> > -    y = np.arange(21)
> > +    y = range(21)
>
> Actually np.as{,any}array handles range objects fine in Py3k.
> But I do need to work on the test.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4573/files#r33704379>.
>